assets/scss/modules/_atomic-class.scss

```diff
@@ -0,0 +1,54 @@
+/* variables margin bottom */
+$dp-m-bottom--lv0: 0;
+$dp-m-bottom--lv1: 6px;
```

Comment:
What do you think about naming it  `$dp-m--lv1`, I mean, all margins _lv1_ will have 6px, right? or does it care if it is a top, botton, left or right marging?

Reply:
No, los margenes son top, right, bottom, left y en algunos casos el margen es para los cuatro valores. por eso $dp-m-bottom es diferente a $margin--lv0 (que me olvide de agregar)

Reply:
Me parece que no. Mejor lo chateamos.

Reply:
Wouldn't it be better to create a mixin that can handle margin/padding positioning and value instead of creating each of the classes manually?

Reply:
I do not know how to do that @damianmuti , do you have any examples at hand?

Reply:
Regarding the mixings, in this case, I think that right now it is enough simple. So, if you have time to talk about it and use the mixings, it is fine. If not, keep this improvement for the future.

Maybe @damianmuti could generate a little PR with the refactoring after your merge.